zipline/protocol.py

```diff
@@ -270,7 +272,7 @@ def __contains__(self, name):
     def __repr__(self):
         return "SIDData({0})".format(self.__dict__)
 
-    def _get_buffer(self, bars, field='price'):
+    def _get_buffer(self, bars, field='price', raw=False):
```

Comment:
This isn't an internal repo. I get burned all the time when people change "private" methods in open source projects. And those are usually just signature or whatever changes, not return a wholly different data structure.